sql/pom.xml

```diff
@@ -255,6 +260,140 @@
           </execution>
         </executions>
       </plugin>
+
+      <plugin>
```

Comment:
nit - are these plugins arranged in order of execution? It will be nice if they are since following the code becomes easier.